app/views/root/_filter.html.erb

```diff
@@ -0,0 +1,81 @@
+<%
+  assignees = [{text: "All assignees", value: ""}]
+  content_types = []
+  states = []
+
+  @presenter.available_users.map do | user |
+    if user.id.to_s == @presenter.assigned_to
+      assignees << {text: user.name, value: user.id, selected: "true"}
+    else
+      assignees << {text: user.name, value: user.id}
+    end
+  end
+
+  format_filter_selection_options.map do | format |
+    if format[1] == @presenter.content_type
+      content_types << {text: format[0], value: format[1], selected: "true"}
+    else
+      content_types << {text: format[0], value: format[1]}
+    end
+  end
+
+  state_names.collect do |scope, status_label|
```

Comment:
I'd be tempted to move this logic to the `FilteredEditionsPresenter`.

Reply:
We should probably also be consistent in whether we use `map` or `collect` (they appear to be aliases for the same method as far as I can tell).